modules/apps/dynamic-data-mapping/dynamic-data-mapping-uad/src/main/java/com/liferay/dynamic/data/mapping/uad/helper/DDMUADHelper.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.dynamic.data.mapping.uad.helper;
+
+import com.liferay.dynamic.data.mapping.model.DDMFormInstance;
+import com.liferay.dynamic.data.mapping.uad.constants.DDMFormInstanceUADConstants;
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.security.xml.SecureXMLFactoryProviderUtil;
+import com.liferay.portal.kernel.util.Time;
+
+import java.io.IOException;
+import java.io.StringReader;
+
+import java.util.Date;
+import java.util.Map;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.osgi.service.component.annotations.Component;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Node;
+
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
+
+/**
+ * @author Gabriel Ibson
+ * @author Marcos Martins
+ */
+@Component(immediate = true, service = DDMUADHelper.class)
```

Comment:
@marcosapmf, is this necessary after last Brian's comment?

Reply:
@natocesarrego I removed this on rebase. It went back because of there are some conflicts on process probably. I'm going to remove it again